bindings/matrix-sdk-ffi/src/notification_service.rs

```diff
@@ -0,0 +1,45 @@
+use std::sync::Arc;
+
+use crate::TimelineItem;
+
+#[allow(dead_code)]
```

Comment:
Is it deadcode?

Reply:
yeah this is more of stub for the future